src/rpc/rawtransaction.cpp

```diff
@@ -60,6 +59,17 @@ void ScriptPubKeyToJSON(const CScript& scriptPubKey, UniValue& out, bool fInclud
     out.push_back(Pair("addresses", a));
 }
 
+namespace {
+    void fillStdFields(UniValue & out, CTxIn const & txin) {
+        out.push_back(Pair("txid", txin.prevout.hash.GetHex()));
```

Comment:
I think for sigma spends prevout == null. This must crash. No?

Reply:
It's filled with zerroes. It is not a pointer in CTxIn class...